services/minespace-web/src/tests/components/dashboard/mine/projects/project/Projects.spec.tsx

```diff
@@ -1,25 +1,39 @@
 import React from "react";
-import { Projects } from "@/components/dashboard/mine/projects/Projects";
 import * as MOCK from "@mds/common/tests/mocks/dataMocks";
 import { render } from "@testing-library/react";
 import { ReduxWrapper } from "@/tests/utils/ReduxWrapper";
-import { PROJECTS } from "@mds/common/constants/reducerTypes";
+import { NOTICE_OF_WORK, PROJECTS } from "@mds/common/constants/reducerTypes";
 import { SidebarProvider } from "@mds/common/components/common/SidebarWrapper";
 import { BrowserRouter } from "react-router-dom";
+import ViewProjects from "@/components/dashboard/mine/projects/ViewProjects";
 
 const initialState = {
   [PROJECTS]: { projects: MOCK.PROJECTS.records },
+  [NOTICE_OF_WORK]: { noticeOfWorkList: MOCK.NOW.applications },
 };
 
 const mine = MOCK.MINES.mines[MOCK.MINES.mineIds[0]];
 
 describe("Projects", () => {
-  it("renders properly", () => {
+  it("renders major mines application tab properly", () => {
     const { container } = render(
       <BrowserRouter>
         <ReduxWrapper initialState={initialState}>
-          <SidebarProvider value={{ mine } as any}>
-            <Projects />
+          <SidebarProvider value={{ mine, subTab: "major-mine" } as any}>
+            <ViewProjects />
```

Comment:
Bit of a nitpick, but if the component is renamed, the test/snapshot should too so it's easy to point to test failures. But in this case I think it's better to revert the name of the component and name the NoW component to match the name of the file.